requirements_test.txt

```diff
@@ -0,0 +1,15 @@
+# If you add/remove requirements here, update ``requires`` in ``setup.py`` (root directory)
+pyramid==1.5.1
+pyramid_mako==1.0.2
+WebError==0.10.3
+pytest==2.6.3
+webtest==2.0.15
+simplejson==3.5.3
+numpy==1.8.1
+scipy==0.14.0
+colander==1.0b1
+sphinx==1.2.2
+breathe==1.2.0
+sphinxcontrib-httpdomain==1.2.1
+sphinx_rtd_theme==0.1.6
+cma
```

Comment:
Also we need to review `requirements.txt` to wipe out any redundant dependencies and pin version.